### PR TITLE
AP-SDK

### PR DESCRIPTION
**Pull request by ninjeeter**

### CODE REVIEW

File: src/concepts/essentials/workflows/sdk.md [hunk 1]
```diff
@@ -0,0 +1,1177 @@
...
+  title: "Title",
+  description: "Description",
+  reporter: "Reporter",
+  request,
+});
+```
+
+:::
+
+#### `requests: ReequestsSDK;`
```
Comment (Sytten) [inline src/concepts/essentials/workflows/sdk.md:883]: `ReequestsSDK` -> `RequestsSDK`

File: src/concepts/essentials/workflows/sdk.md [hunk 2]
```diff
@@ -0,0 +1,1177 @@
...
+  .catch((err) => {
+    sdk.console.error(err);
+  });
+```
+
+:::
+
+#### `asString(array: Bytes): string;`
+
+- A helper function that converts Bytes (`string/Array<number>/Uint8Array`) to a string, replacing unprintable characters with `�`.
```
Comment (Sytten) [inline src/concepts/essentials/workflows/sdk.md:906]: Thats the only tip I would keep since its a direct method call

**Review by Sytten — CHANGES_REQUESTED** [review]
Good first pass, lets talk about it in Discord 👍

File: src/concepts/essentials/workflows/sdk.md (hunk 2 above)
Comment (ninjeeter) [reply src/concepts/essentials/workflows/sdk.md:906]: I do feel like the send() method example is beneficial. At least for newcomers. Which I feel like it's mostly going to be newcomers reading this page in general lol.

File: src/concepts/essentials/workflows/sdk.md [hunk 3]
```diff
@@ -0,0 +1,1177 @@
...
+
+#### `request: Request;`
+
+- `request` stores the [Request](#request) object.
+
+#### `response: Response;`
+
+- `response` stores the [Response](#response) object.
+
+## RequestsSDK
```
[the remark this answers is not in the document]
Comment (ninjeeter) [reply src/concepts/essentials/workflows/sdk.md:642]: I am not sure how that would be divided up.

File: src/concepts/essentials/workflows/sdk.md [hunk 4]
```diff
@@ -0,0 +1,1177 @@
+# SDK
```
[the remark this answers is not in the document]
Comment (ninjeeter) [reply src/concepts/essentials/workflows/sdk.md]: Moved it to References.

File: src/concepts/essentials/workflows/sdk.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ninjeeter) [reply src/concepts/essentials/workflows/sdk.md]: Do you want me to reorganize the actual type file based on how the page ends up being organized? I just went line by line according to the file for the documentation.

File: src/concepts/essentials/workflows/sdk.md [hunk 5]
```diff
@@ -0,0 +1,1177 @@
...
+
+The `typing.d.ts` file is the interface for Caido's SDK/API. Within it you can find the available methods. This SDK provides a simple way to interact with the Caido platform using your TypeScript or JavaScript code.
+
+::: info
+To view the **typing.d.ts** file - click [here](https://github.com/caido/sdk-workflow/blob/main/src/typing.d.ts).
+
+Or expand the collapsible element located at the bottom of this page.
```
[the remark this answers is not in the document]
Comment (Sytten) [reply src/concepts/essentials/workflows/sdk.md]: Leave it out, its fine

File: src/concepts/essentials/workflows/sdk.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Sytten) [reply src/concepts/essentials/workflows/sdk.md]: So the way I see it
- SDK
  - RequestsSDK
  - FindingsSDK
- Types
  - Body

File: src/concepts/essentials/workflows/sdk.md (hunk 2 above)
Comment (Sytten) [reply src/concepts/essentials/workflows/sdk.md]: Thats fine

**Review by Sytten — APPROVED** [review]

**Pull request merged**